### ISSUE TITLE
Alternative track sources can't find a single correct track even though it's the first search result on YT

### ISSUE THREAD

**Issue by 256shadesofgrey** [KRTirtho/spotube#1447]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When I try to play ["Can't Find It" from the band "Love"](https://youtu.be/Ly98MkAVVPs), the default track being played is just some video of some guy talking about finding love. And when I press "alternative track sources", almost all suggestions are videos that have nothing to do with any kind of music at all, and the songs that it does suggest are from other bands, and don't even have the same title (it suggests a few songs called "Can't Find **Love**", not "Can't Find **It**").

### Expected Behavior

Alternative track sources should be at least as good as generic youtube search.

### Steps to reproduce

1. Add the song "Can't Find It" from the band "Love" to a playlist on spotify.
2. Try to play the song through spotube.

### Operating System

Android

### Spotube version

v3.6.0

### Installation source

GitHub Releases (Binary)

### Additional information

This issue is different from #883, because the song does exist on youtube, even in [multiple versions](https://youtu.be/x0oLO_Es7_s). And when searching for "love can't find it", the way I would normally do, it's the first search result, which for some reason isn't shown in alternative track sources in the app. And even when searching for "can't find it - love" the way it's written after pressing the search button in alternative track sources, it's the 2nd search result.

In other words, it seems like the complicated scoring algorithm described [here](https://github.com/KRTirtho/spotube/issues/883#issuecomment-1867716054) is in this case causing more problems than it solves, so a fallback to a generic search would be helpful, for example in the form of a "more" button at the bottom of the alternative track selection list, or by mixing in actual YT search results in the suggestions by default. So would an option for formatting the search string, which if I remember correctly, used to be there in older versions of the app.

### Self grab

- [ ] I'm ready to work on this issue!
